meta/Cargo.toml

```diff
@@ -41,14 +41,17 @@ tracing = { workspace = true }
 tracing-futures = { workspace = true }
 uuid = { workspace = true }
 walkdir = { workspace = true }
+psutil = { workspace = true }
```

Comment:
These two dependencies are optional, please add `optional = true` option, and open them in feature `meta_e2e_test`.